glyphs2fontir/src/source.rs

```diff
@@ -833,6 +897,8 @@ impl Work<Context, WorkId, Error> for KerningInstanceWork {
 #[derive(Debug)]
 struct GlyphIrWork {
     glyph_name: GlyphName,
+    // If present, we are building a bracket glyph.
+    base_name: Option<GlyphName>,
```

Comment:
Suggest a name that's a bit clearer, e.g. bracket_glyph_parent or something. base_name is a bit vague.